TombEngine/Scripting/Internal/TEN/Strings/StringsHandler.cpp

```diff
@@ -82,6 +89,20 @@ void StringsHandler::ShowString(DisplayString const & str, sol::optional<float>
 	it->second.m_isInfinite = !nSeconds.has_value();
 }
 
+bool StringsHandler::isStringDisplaying(DisplayString const& str)
```

Comment:
Please capitalise the first letter in the function name, and make the function `const` (if possible).